### ISSUE TITLE
applyScopes() never called

### ISSUE THREAD

**Issue by hrabbit** [matchory/elasticsearch#103]
Hi all,

I might be missing something while using this library, but I'll report it here in the event that it's more global.

While performing a local scoped query, I can perform my queries without issue.

```
...
protected static function scopeFoo(Query $query) {
 $query->where('foo', 'bar');
}
```

```
$x = Model::Foo();
```
This returns a query, as expected, containing a bool check for "foo" = "bar"
```
...
"body" => array:1 [▼
    "_source" => array:2 [▼
      "includes" => [],
      "excludes" => [],
      "query" => array:1 [▼
        "bool" => array:1 [▼
          "filter" => array:1 [▼
            0 => array:1 [▼
              "term" => array:1 [▼
                "foo" => "bar"
              ]
            ]
          ]
        ]
      ]
...
```

However, trying to apply a global scope of the same thing, results in the same query missing the bool filter.

```
protected static function booted(): void
 {
        static::addGlobalScope('fooIsBar', function (Query $query)
        {
            $query->where('foo','bar');
        });
}
```
Yields a result of
```
"body" => array:1 [▼
    "_source" => array:2 [▼
      "includes" => [],
      "excludes" => []
    ]
  ]
...
```

After poking around in the code for a while, it appears that Concerns\AppliesScopes::applyScopes() is never actually called.

I tested this simply by running

```
$x = Model::applyScopes();
```

Which did return the query and included the global scope.

I can possibly try and correct this but I'm unsure where applyScopes is meant to be called within the module.

Hopefully there is enough clarity above.

**Comment by Radiergummi** [comment]
Hi @hrabbit, thank you for the detailed report! This indeed looks like it should work. I'm going to take a look today 👀

**Comment by Radiergummi** [comment]
Fix is released as [2.6.2](https://packagist.org/packages/matchory/elasticsearch#2.6.2)
